frontend/src/config/config.ts

```diff
@@ -1,3 +1,3 @@
-const baseURL = process.env.NODE_ENV == 'production' ? '/' : 'http://localhost:8080';
+const baseURL = process.env.NODE_ENV === 'production' ? '/' : 'http://localhost:8080';
```

Comment:
While this is a reasonable change, it'll make more sense to have only the thing changed that is mentioned
```suggestion
const baseURL = process.env.NODE_ENV == 'production' ? '/' : 'http://localhost:8080';
```